helm/mail/templates/configmap.yaml

```diff
@@ -13,6 +13,9 @@ data:
   {{- range $key, $value := .Values.config.postfix }}
   POSTFIX_{{ $key }}: {{ $value | quote }}
   {{- end }}
+  {{- if .Values.metrics.enabled }}
+  POSTFIX_maillog_file: {{ .Values.metrics.maillog }}
+  {{- end }}
```

Comment:
I don't know how I feel about this ATM. It gets the job done but it can break [the principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) for users who would like to override `maillog_file` themselves -- we've already had cases of users who use this feature.

Reply:
As long as there is some way to have a log file written for the exporter to pick up it should be fine.
The exporter will not work without the log file, so as long as there is a good way where the maillog_file overwrite is being done well where everyone is happy it should be good.

Reply:
I know. Let me think about it, though if we don't find a better solution, we should add it to the documentation at least.